.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Lint and Test
+
+# Run this when someone pushes a branch
+on: [push]
+
+jobs:
+  # Get the versions of node and npm as defined in the package.json engines
+  get-versions:
+    runs-on: ubuntu-latest
+
+    outputs:
+      node_version: ${{ steps.node_version.outputs.prop }}
+      npm_version: ${{ steps.npm_version.outputs.prop }}
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Read node version from package.json
+      id: node_version
+      uses: notiz-dev/github-action-json-property@v0.1.0
+      with:
+        path: "package.json"
+        prop_path: "engines.node"
+
+    - run: |
```

Comment:
**[learning]** what does the `|` do here?

Reply:
It's essentially a concatenation operator for stringing together bash commands on separate lines.  It's being used here just for formatting so that the command can be on its own line for readability.